src/test-e2e/cloud-foundry/cf-level/edit-org-e2e.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { by, element, promise, protractor } from 'protractor';
+
+import { e2e } from '../../e2e';
+import { CFHelpers } from '../../helpers/cf-helpers';
+import { ConsoleUserType, E2EHelpers } from '../../helpers/e2e-helpers';
+import { CfOrgLevelPage } from '../org-level/cf-org-level-page.po';
+import { OrgFormPage } from './org-form-page.po';
+
+describe('Edit Organization', () => {
```

Comment:
To save on e2e duration I'd recommend testing edit at the same time as create. This would mean edits are in an org created in the same session but that's preferable than the additional time it takes to setup and run a new test.